parm/soca/varchange/soca_2cice_antarctic.yaml

```diff
@@ -18,26 +18,26 @@ variable change:
     min hsno: 0.1
   domain: antarctic
   cice background state:
-    restart: $(ICE_RST)
+    restart: {{ ICE_RST }}
```

Comment:
Do we also need to add beginning and ending single quote, `'`?  That is, `'{{ ICE_RST }}'`?

Reply:
I think, in general, YAML assumes it is a string if it is something 'non-numeric', so all the quotes would do here is make it explicitly a string.

Reply:
OK.  I see.  So the more important consideration is to ensure `{{ numeric_value }}` does not get quoted.  Quotes around non-numeric strings are optional.  That noted, should we go ahead and add quotes around non-numeric strings for the sake of consistency. I'm thinking about developers (myself included!) who use existing yamls as examples when developing new yamls.

Reply:
Yeah I think being explicit is generally a good idea. Do we want that here or in a separate PR?

Reply:
I'm inclined to update the yamls needing this change now.  If this is too many yamls to change at once, we could limit quoting to only the yamls in this PR which need it.  The remaining yamls could be captured in a subsequent PR.

Reply:
I can go through the yamls in this PR and add quotes around non-numeric `{{ foo }}`.

Reply:
No need, I can do that this afternoon.

Reply:
OK. Thank you!

Reply:
@RussTreadon-NOAA I think I've updated appropriately. I will note that there are places where the variable is non-templated and is an implicit string rather than a explicit string.

For example, `'{{ ATM_WINDOW_LENGTH }}'` will end up being `'PT6H'` but there are YAMLs that just say something like `step: PT3H`. I guess this is okay (at least for now)?

Reply:
Works for me.